lld/COFF/InputFiles.cpp

```diff
@@ -260,6 +297,8 @@ SectionChunk *ObjFile::readSection(uint32_t sectionNumber,
     guardEHContChunks.push_back(c);
   else if (name == ".sxdata")
     sxDataChunks.push_back(c);
+  else if (isArm64EC(getMachineType()) && name == ".hybmp$x")
```

Comment:
Will the section name always have the `$x` suffix, or should we have something like `name.substr(0, name.find('$')) == ".hybmp"`?

Reply:
I checked with MSVC, it seems to apply the special treatment only for that exact name. Things like ".hybmp", ".hybmp$xy" or ".hybmp$y" don't get that treatment.